distributed/protocol/numpy.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import print_function, division, absolute_import
+
+import sys
+
+import numpy as np
+
+try:
+    import blosc
+    n = blosc.set_nthreads(2)
```

Comment:
We want some benefit from threads, but don't want to take them all (dask itself does parallelism)  Two seemed like a decent default (more decent than "all").  Do you have other thoughts on what this should be?